### PR TITLE
Introduce state

### PR DESCRIPTION
**Pull request by KateHavrenko**
Please review this, Kelly!

### CODE REVIEW

File: src/Details.js [hunk 1]
```diff
@@ -1,11 +1,29 @@
-import React from 'react';
+import React, {Component} from 'react';
```
Comment (smellyung) [inline src/Details.js]: Spacing between curly braces and `Component`, i.e `{ Component }` :)

File: src/Details.js [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
-        <div>
-            <p>Hello, this will be the details page for each Movie & TV show :)</p>
-            <Link to='/'>Back to home</Link>
-        </div>
-    );
+export default class Details extends Component {
+    constructor() {
+        super();
+        this.state = {
+            welcomeMessage: 'Welcome to the details, WIP :)'
```
Comment (smellyung) [inline src/Details.js]: you can remove the space between `this.state = {` and the welcomeMessage

Comment (smellyung) [reply src/Details.js]: Seems to be a issue with github showing the code, here it seems fine but when 'view changes' clicked theres a space between these two lines of code. No need for changes here :)

**Comment by GrahamHewett** [comment]
Great! Let's merge!

**Pull request merged**